kubeflow/tf-serving/tf-serving.libsonnet

```diff
@@ -358,7 +363,8 @@
           spec+: {
             containers: [
               $.gcpParts.tfServingContainer,
-              if $.util.toBool($.params.deployHttpProxy) then
+              if $.util.toBool($.params.deployHttpProxy) &&
+                 $.util.toBool($.params.exposeRestApi) == false then
```

Comment:
This will break the existing behavior: set deployHttpProxy to true won't deploy http proxy.

So in test http proxy is not deployed: https://github.com/kubeflow/kubeflow/blob/master/components/k8s-model-server/images/releaser/components/workflows.libsonnet#L95